src/commands/stats.ts

```diff
@@ -0,0 +1,57 @@
+import { botID } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/module/client.ts";
+import { botCache } from "../../mod.ts";
+import { cache } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/utils/cache.ts";
+import { avatarURL } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/handlers/member.ts";
+import { sendMessage } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/handlers/channel.ts";
+
+botCache.commands.set(`stats`, {
+  name: `stats`,
+  guildOnly: true,
+  execute: (message, _args, guild) => {
+    let botMember = guild?.members.get(botID);
+
+    if (!botMember) return;
+
+    let memberCount = 0;
+
+    cache.guilds.forEach((guild) => {
```

Comment:
There are a couple things i would recommend we optimize here.
1. This is only listed cached members. Sometimes, it is useful to know how many total members the bot has. We should also have that in this command.
2. forEach isn't good to use here. It just runs this function for all guilds in the background while the rest of the code will continue. We don't want that. We don't want it to send a response without first finishing this math. Instead, we will use a normal for loop(because we have 2 variables).

```ts
let totalMemberCount = 0;
let cachedMemberCount = 0;

for (const guild of cache.guilds.values()) {
    totalMemberCount += guild.memberCount;
    cachedMemberCount += guild.members.size;
}
```